### PR TITLE
feature: guest io throttle

### PR DESCRIPTION
**Pull request by wanyaoqi**
**这个 PR 实现什么功能/修复什么问题**:
feature: guest io throttle
**是否需要 backport 到之前的 release 分支**:
release/2.11

/area region host
/cc @swordqiu @zexi

### CODE REVIEW

File: pkg/compute/models/guest_actions.go [hunk 1]
```diff
@@ -3432,3 +3432,47 @@ func (guest *SGuest) PerformChangeOwner(ctx context.Context, userCred mcclient.T
...
 }
+
+func (self *SGuest) AllowPerformIoThrottle(ctx context.Context,
+	userCred mcclient.TokenCredential,
+	query jsonutils.JSONObject,
+	data jsonutils.JSONObject) bool {
+	return self.IsOwner(userCred) || db.IsAdminAllowPerform(userCred, self, "io-throttle")
+}
+
+func (self *SGuest) PerformIoThrottle(ctx context.Context, userCred mcclient.TokenCredential, query jsonutils.JSONObject, data jsonutils.JSONObject) (jsonutils.JSONObject, error) {
```
Comment (ioito) [inline pkg/compute/models/guest_actions.go]: 最好顺便加下api文档

Comment (wanyaoqi) [reply pkg/compute/models/guest_actions.go]: ok

**Pull request merged**